mule-module-apikit/src/test/java/org/mule/module/apikit/parameters/ParametersTestCase.java

```diff
@@ -58,7 +56,7 @@ public void invalidEnumHeaderProvided() throws Exception
     {
         given().header("one", "invalid")
                 .expect().response().statusCode(400)
-                .body(is("Invalid value 'invalid' for header one. Value must be one of [foo, bar]"))
+                .body(is("Invalid value 'invalid' for Parameter one. Value must be one of [foo, bar]"))
```

Comment:
It seems we are losing some info in the error message, is there any chance to keep that info?